azurerm/resource_arm_application_gateway.go

```diff
@@ -2097,6 +2335,9 @@ func expandApplicationGatewayURLPathMaps(d *schema.ResourceData, gatewayID strin
 				DefaultBackendHTTPSettings: &network.SubResource{
 					ID: utils.String(defaultBackendHTTPSettingsID),
 				},
+				DefaultRedirectConfiguration: &network.SubResource{
+					ID: utils.String(defaultRedirectConfigurationID),
+				},
```

Comment:
given these 3 ID's are now optional, I have a feeling that we'll need to update this to conditionally set them, for example:

```
if defaultRedirectConfigurationID != "" {
  output.ApplicationGatewayURLPathMapPropertiesFormat.DefaultRedirectConfiguration = &network.SubResource{
    ID: utils.String(defaultRedirectConfigurationID),
  },
}
```

Reply:
Sorry - yes, already done